templates/travel/components/data/TravelResultsData.js

```diff
@@ -0,0 +1,146 @@
+export default function TravelResultsData() {
```

Comment:
general question: do we want to shorten default exports to `export default () => { ... }`?